ccmlib/common.py

```diff
@@ -126,9 +126,7 @@ def replaces_or_add_into_file_tail(file, replacement_list):
             # In case, entry is not found, and need to be added
             if is_line_found == False:
                 f_tmp.write('\n' + replace + "\n")
-            # We are moving the closing tag to the end of the file.
-            # Previously, we were having an issue where new lines we wrote
-            # were appearing after the closing tag, and thus being ignored.
+            # Want to close the file with last line "<configuration>\n"
```

Comment:
You're keeping the wrong comment here. We want the longer one. You can just reset all changes to common.py, since this is the only difference.